packages/leveldb/src/leveldb/instance.ts

```diff
@@ -184,8 +188,16 @@ export class RxStorageIntanceLevelDB<RxDocType> implements RxStorageInstance<
       upperBound
     );
     const indexName = getIndexName(index);
+    // => '_meta.lwt,key'
+    // QUESTION: this doesn't seem to map to any of the indexes being entered?
```

Comment:
We're seeing some weirdness where the `queryPlan` generates index name which is then used by `getIndex` and gets nothing ... looking at what's going in with `setIndex` is seems there is some mis-alignment (i.e. no indexes are being set with value `_meta.lwt,key` .... 

Pausing here for today - the next questions are:
- why misalignment
- what should `setIndex` `getIndex` be doing? (where's the definition)
- ....?

Reply:
When you have a query, the storage should check if the current query is satisfied by an index and get the index instead of listing all the rows.

Its done for efficiency, the indexes are just faster to query than whole table if that makes sense.

Reply:
Yeah I know what an index is, have used them a bunch.

What I'm confused about is why the `setIndex` and `getIndex` look so wrong